lib/gmo/http_services.rb

```diff
@@ -74,10 +74,8 @@ def self.create_http(server, options)
             else
               http = Net::HTTP.new server, 443
             end
-            if options[:timeout]
-              http.open_timeout = options[:timeout]
-              http.read_timeout = options[:timeout]
-            end
+            http.open_timeout = GMO::Const::HTTP_READ_TIMEOUT
```

Comment:
This means that we will now ignore `:timeout` received in `options`. Won't this change existing behavior?